src/components/Assets/AssetTable.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { handleTableDataStringLength, formatDataTitles } from "./helpers";
+
+const AssetTable = ({ fields, data }) => {
+  const tableHeaders = fields.map(field =>
+    formatDataTitles(Object.keys(field))
+  );
+  const fieldIds = fields.map(field => {
+    return Object.values(field)[0];
+  });
+  // TODO add responsive font size (or rely on side-scroll?)
+  // TODO fix whitespace text node console warning
+  // TODO handle URLs returned from Knack - make URLs set state to change current asset?
```

Comment:
It would be honorable work to parse those fields for html or use the `_raw` field values, but seems like too much work for our situation. We could revisit this if we hear complaints from user, or go ahead and backlog it for the future.

Reply:
👍